library/gradle

```diff
@@ -34,10 +26,15 @@ Tags: 5.4.1-jdk11-slim, 5.4-jdk11-slim, jdk11-slim
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
 Directory: jdk11-slim
 
-Tags: 5.4.1-jre11, 5.4-jre11, jre11
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-Directory: jre11
-
 Tags: 5.4.1-jre11-slim, 5.4-jre11-slim, jre11-slim
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
 Directory: jre11-slim
```

Comment:
Looks like this entry should go away too? (no `jre11-slim` directory in https://github.com/keeganwitt/docker-gradle that I can see)